the-collab-lab/tcl-30-whats-near-me
# [TCL30-54] Reusable Button Components

## Description

As a user,
I would like the buttons to be unified across the application
So that I understand and am comfortable with what functions they perform.

## Related Issue

[TCL30-54](https://the-collab-lab.atlassian.net/browse/TCL30-54)

## Acceptance Criteria

- Create `Button` component.
- We should show **icons as buttons** and **`NavLink` as buttons**. 😄 .

**Icon as buttons**
* close icon
* fly to me icon
* search place icon
* share location icon

**`NavLink` as buttons**
* list link
* map link
* settings link
* help link

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
| ✓   | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|     | :link: Update dependencies |
|     | :scroll: Docs              |

## Updates

### Before

![image](https://user-images.githubusercontent.com/8689897/131715248-a484e831-0a4d-4932-8efc-3fe4d4124f98.png)

![image](https://user-images.githubusercontent.com/8689897/131715863-a5f9396a-72b6-4430-afe1-c7ef7bfaeb9d.png)


### After

![image](https://user-images.githubusercontent.com/8689897/131715385-d4191ee9-65b4-4fad-a46b-750891e13858.png)

## Testing Steps / QA Criteria

1. Checkout to `jw-reusable-button-components`  branch.
2. You can see our `Button` component in `src/components`
3. If you can see as buttons look, you might visit `/ui`.
